.github/workflows/pre-commit.yml

```diff
@@ -0,0 +1,14 @@
+name: pre-commit
```

Comment:
I am not very familiar with github actions, how does this work? Does it automatically run the pre-commit on a PR and checks if it has been formatted using it?

Reply:
yes that is the idea, but @psss needs to enable `Actions` as project owner

Reply:
not sure though why it does not run now

Reply:
I guess that is becuase we did not yet onboard at all ...